phoenicis-dist/src/flatpak/org.phoenicis.javafx.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=Phoenicis JavaFX
+Comment=Front-end for Wine
+Exec=phoenicis-javafx %U
+StartupNotify=true
+Terminal=false
+Type=Application
+MimeType=application/x-ms-dos-executable;application/x-msi;application/x-ms-shortcut;
+Icon=org.phoenicis.javafx
+Categories=Game
+Keywords=Phoenicis;
```

Comment:
PlayOnLinux, Wine

Reply:
I'm not sure how the keywords need to be split.
If they are split with a `;`, the separator between PlayOnLinux and Wine is wrong.

Reply:
Fixed.